src/main/java/frc/robot/commands/arm/ExtendToPosition.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands.arm;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.FourBarArm;
+
+public class ExtendToPosition extends CommandBase {
+  FourBarArm m_arm;
+  PIDController m_pid;
+  double m_armSetpoint;
+  public ExtendToPosition(FourBarArm arm, PIDController pid, double setpoint) {
+    m_arm = arm;
+    m_pid = pid; 
+    m_armSetpoint = setpoint;
+  }
+  @Override
+  public void initialize() {
+    m_pid.reset();
+    m_arm.setEncoderPosition(0);
+  }
+  @Override
+  public void execute() {
+    m_arm.setMotor(m_pid.calculate(m_arm.getEncoderPosition(), m_armSetpoint));
+  }
+  @Override
+  public boolean isFinished() {
+    double error = m_armSetpoint - m_arm.getEncoderPosition();
+    return error <= -1 || error >= -1; // error values TBD
```

Comment:
Use Math.abs(error)? Also, introduce a constant kMaxFinalError, so it will look like `return Math.abs(error) < MaxFinalError`

Reply:
Based on Jerry's comment below, this is no longer needed.